cli/packages/cmd/run.go

```diff
@@ -149,7 +175,15 @@ var runCmd = &cobra.Command{
 
 		log.Debug().Msgf("injecting the following environment variables into shell: %v", env)
 
-		Telemetry.CaptureEvent("cli-command:run", posthog.NewProperties().Set("secretsCount", len(secrets)).Set("environment", environmentName).Set("isUsingServiceToken", infisicalToken != "").Set("single-command", strings.Join(args, " ")).Set("multi-command", cmd.Flag("command").Value.String()).Set("version", util.CLI_VERSION))
+		Telemetry.CaptureEvent("cli-command:run",
+			posthog.NewProperties().
+				Set("secretsCount", len(secrets)).
+				Set("environment", environmentName).
+				Set("isUsingServiceToken", token.Type == "service-token").
```

Comment:
when you run the `run` command, it will throw nill pointer error

Reply:
```
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x2 addr=0x8 pc=0x104dd1a20]

goroutine 1 [running]:
github.com/Infisical/infisical-merge/packages/cmd.glob..func16(0x1400017ad00?, {0x1400025db40, 0x1, 0x2})
        /Users/maidulislam/Desktop/infisical/cli/packages/cmd/run.go:183 +0xd30
github.com/spf13/cobra.(*Command).execute(0x1053e9ca0, {0x1400025db20, 0x2, 0x2})
        /Users/maidulislam/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:920 +0x654
github.com/spf13/cobra.(*Command).ExecuteC(0x1053e96e0)
        /Users/maidulislam/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:1044 +0x320
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/maidulislam/go/pkg/mod/github.com/spf13/cobra@v1.6.1/command.go:968
github.com/Infisical/infisical-merge/packages/cmd.Execute()
        /Users/maidulislam/Desktop/infisical/cli/packages/cmd/root.go:32 +0x24
main.main()
        /Users/maidulislam/Desktop/infisical/cli/main.go:16 +0x178
exit status 2
```

Reply:
Strange, I'm unable to reproduce this. Can you include reproduction steps please?